.travis/install_test.sh

```diff
@@ -4,12 +4,7 @@ set -ev
 mkdir --parents "${HOME}/bin"
 
 # PHPUnit install
-if [ "${TRAVIS_BRANCH}" = 'master' ]; then
-    PHPUNIT_VERSION=7
-else
-    PHPUNIT_VERSION=5.7
-fi
-wget "https://phar.phpunit.de/phpunit-${PHPUNIT_VERSION}.phar" --output-document="${HOME}/bin/phpunit"
+wget "https://phar.phpunit.de/phpunit-7.phar" --output-document="${HOME}/bin/phpunit"
```

Comment:
This has to be done again with Travis… maybe find another condition based on the version of PHP?